spec/services/twitter_service_spec.rb

```diff
@@ -7,6 +7,113 @@
       stub_request(:post, /api\.twitter\.com/).to_return(:status => 200, :body => '{ "id": 243145735212777472, "text": "hello world" }')
     end
 
+    let(:hangout) { FactoryGirl.create(:event_instance, updated: '10:00 UTC', hangout_url: nil, yt_video_id: nil) }
+
+    context 'pair programming tweet' do
+      before(:each) do
+        hangout.category = 'PairProgramming'
+        hangout.participants = {"0" => {"id" => "xxx.id.google.com^xxx", "hasMicrophone" => "true",
+                                        "hasCamera" => "true", "hasAppEnabled" => "false",
+                                        'isBroadcaster' => "false", "isInBroadcast" => "true",
+                                        "displayIndex" => "0", "person" =>
+                                            {"id" => "xxx", "displayName" => "Foo Bar",
+                                             "image" => {"url" => ".../s96-c/photo.jpg"},
+                                             "fa" => "false"}, "locale" => "en", "fa" => "false"},
+                                "1" => {"id" => "xxx.id.google.com^xxx", "hasMicrophone" => "true",
+                                        "hasCamera" => "true", "hasAppEnabled" => "false",
+                                        "isBroadcaster" => "false", "isInBroadcast" => "true",
+                                        "displayIndex" => "1", "person" =>
+                                            {"id" => "xxx", "displayName" => "Bar Foo",
+                                             "image" => {"url" => ".../s96-c/photo.jpg"},
+                                             "fa" => "false"}, "locale" => "en", "fa" => "false"},
+                                "2" => {"id" => "xxx.id.google.com^xxx", "hasMicrophone" => "true",
+                                        "hasCamera" => "true", "hasAppEnabled" => "true",
+                                        "isBroadcaster" => "true", "isInBroadcast" => "true",
+                                        "displayIndex" => "2", "person" =>
+                                            {"id" => "xxx", "displayName" => "John Doe",
+                                             "image" => {"url" => ".../s96-c/photo.jpg"}, "fa" => "false"},
+                                        "locale" => "en", "fa" => "false"}}
```

Comment:
non-blocking issue, but it looks like the participants fixtures are duplicated here, perhaps you could move this into a `let` block a little higher up, since they take up quite a bit of space :grinning: